Pinta.Core/Managers/IChromeManager.cs

```diff
@@ -30,7 +30,61 @@
 
 namespace Pinta.Core;
 
-public sealed class ChromeManager
+public interface IChromeManager
```

Comment:
Naming this `IChromeService` seems more consistent with the other interfaces?

Reply:
I've renamed it to `IChromeService`